src/types/primitive.rs

```diff
@@ -4,10 +4,11 @@ use crate::ops::AliasDecl;
 
 use super::{CustomType, FunctionType, TypeBound};
 
+#[allow(missing_docs)]
```

Comment:
i think this should have a docstring, at least for the enum itself, I think you can place the `allow` within the enum definition to allow missing docs on variants

Reply:
Done